src/translations.php

```diff
@@ -72,6 +72,15 @@
         "Longest Week Streak" => "Най-дълга седмична серия",
         "Present" => "Сега",
     ],
+    "bho_IN" => [
```

Comment:
I think country specification is not needed
```suggestion
    "bho" => [
```

Reply:
Thank you for the clarification! I’ll update the language code to 'bho' and push the changes shortly.